red-hat-data-services/jupyterhub-odh
# WIP: [RHODS-3183] Prevent default notebook selection from using disabled notebook image

- [ ] JIRA link(s): [RHODS-3183|https://issues.redhat.com/browse/RHODS-3183]

TEST STEPS
===
1. Deploy JH from the live build
2. Create a notebook image imagestream and buildconfig that will create a JH notebook image. CUDA notebooks can be used for testing
3. Wait for the example notebook to finish building and spawn a JH notebook when it is finished
4. Shutdown the user notebook and start a rebuild of that user notebook
5. Navigate to the JH Spawner UI and confirm that the user previous notebook selection is disabled and NOT selected
